pkg/controller/vitesscluster/reconcile_vtctld.go

```diff
@@ -18,6 +18,7 @@ package vitesscluster
 
 import (
 	"context"
+
```

Comment:
I assume your IDE did this?

Reply:
Yep. Due to the new sorting stuff we turned on that keeps stdlib as its own group at the top.

Reply:
It's a good thing.